xls/synthesis/timing_characterization_client_main.py

```diff
@@ -54,85 +88,304 @@ def _synth(stub: synthesis_service_pb2_grpc.SynthesisServiceStub,
   return stub.Compile(request)
 
 
-def _run_binop(
-    op: str, kop: str, stub: synthesis_service_pb2_grpc.SynthesisServiceStub
-) -> List[delay_model_pb2.DataPoint]:
-  """Characterize a binary operation via synthesis server.
+def _run_unary_bitwise(
+    kop: str, model: delay_model_pb2.DelayModel,
```

Comment:
use 'op' instead of 'kop'. The 'k' prefix comes from C++ where it is used as a prefix for constants (like 'kAdd').

Reply:
(solved)